src/main/java/org/opensearch/commons/model2/action/ExecuteMonitorRequest.java

```diff
@@ -0,0 +1,33 @@
+package org.opensearch.commons.model2.action;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.action.ActionRequest;
+import org.opensearch.action.ActionRequestValidationException;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.commons.model2.ModelSerializer;
+import org.opensearch.commons.model2.model.Monitor;
+
+import java.io.IOException;
+
+public class ExecuteMonitorRequest extends ActionRequest {
```

Comment:
Can we create a dedicated sub-project (common path) for all alerting related models